### ISSUE TITLE
Zoom in only impacts notes, not sidebar sizing, which is a major usability issue for large and high DPI monitors

### ISSUE THREAD

**Issue by jkterry1** [Automattic/simplenote-electron#1037]
The basically title says it all:

Zooming in zooms in on notes, not the list of notes. For people with 4k monitors, other very large monitors, or vision issues, this can still leave the sidebar unreadable, which is a major impediment to using simplenote.

In case this is a bug and not a missing feature, this occurs on Ubuntu 18.04 and 18.10 on the most recent release of simplenote

**Comment by mirka** [comment]
We plan on having dedicated typography settings for the editor, so the editor font size can be set independently (#274).

**Comment by sprite-1** [comment]
I hope you don't mean like actually changing the font size within the individual notes itself because I use the zoom functions only when I'm writing rough drafts in focus mode. Then i reset it back to normal afterwards. 🤔

**Comment by codebykat** [comment]
Zooming in now correctly zooms in both the sidebar/tag list and the note.
